src/test/regress/expected/multi_hash_pruning.out

```diff
@@ -59,120 +59,120 @@ SET client_min_messages TO DEBUG2;
 -- Check that we can prune shards for simple cases, boolean expressions and
 -- immutable functions.
 EXPLAIN SELECT count(*) FROM orders_hash_partitioned;
-                              QUERY PLAN                              
-----------------------------------------------------------------------
- explain statements for distributed queries are currently unsupported
+                         QUERY PLAN                         
+------------------------------------------------------------
+ explain statements for distributed queries are not enabled
```

Comment:
Is there any reason not to enable this by default? Seems like a pretty pointless complication for users.